dags/veda_data_pipeline/groups/discover_group.py

```diff
@@ -24,11 +24,6 @@ def discover_from_s3_task(event: dict={}, ti=None, payload: dict={}, prev_start_
         **event,
         **payload,
     }
-    # If the DAG is triggered by a schedule add the configuration to DAG run
```

Comment:
Before merging, let's confirm this. I recall this being essential for scheduled DAGs that include a discovery step